src/config/settings/base.py

```diff
@@ -318,9 +320,10 @@
 
 
 # Add additional configuration below:
-# ------------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
```

Comment:
The changes here, and on line 326 seem harmless but also unnecessary and inconsistent with the rest of the file.